pkg/volume/csi/csi_client.go

```diff
@@ -263,3 +248,21 @@ func asCSIAccessMode(am api.PersistentVolumeAccessMode) csipb.VolumeCapability_A
 	}
 	return csipb.VolumeCapability_AccessMode_UNKNOWN
 }
+
+func newGrpcConn(driverName string) (*grpc.ClientConn, error) {
+	if driverName == "" {
+		return nil, fmt.Errorf("driver name is empty")
+	}
+
+	network := "unix"
```

Comment:
What  about defining  `network` as a const at the top of the file? I think it would be more visible.

Reply:
ok

Reply:
I thought about it, the value for network does not need to be global since it should always be "unix" and is only used when constructing the connection in that method.